pkgs/by-name/la/languagetool-rust/package.nix

```diff
@@ -78,11 +85,13 @@ rustPlatform.buildRustPackage rec {
       --zsh <($out/bin/ltrs completions zsh)
   '';
 
-  meta = with lib; {
+  passthru.updateScript = nix-update-script { };
```

Comment:
What's the point of adding this? As I remember, the default `nixpkgs-update` worked well before

Reply:
Oh, that's interesting. I've never used it to generate commits, it looks very convenient. Thanks for sharing!